packages/core/test/combinator/concatMap-test.js

```diff
@@ -96,18 +92,16 @@ describe('concatMap', function () {
 
   it('should dispose all inner streams', function () {
     const values = [1, 2, 3]
-    const spies = values.map(function () {
-      return spy()
-    })
+    const spies = values.map(() => spy())
 
-    const inners = values.map(function (x, i) {
+    const inners = values.map((x, i) => {
```

Comment:
yah wasnt sure if you had some standards on what you wanted to be an arrow expression based on length of definition or if its a one-liner, etc